scripts/export-director-metadata

```diff
@@ -51,6 +51,12 @@ if [ ! -z ${OPSMAN_PRIVATE_KEY:+x} ]; then
   sleep 10s
 fi
 
+# Set NO_PROXY for BOSH Director
+if [ ! -z ${SET_NO_PROXY:+x} ] && [ $SET_NO_PROXY = true ]; then
+    export NO_PROXY="$NO_PROXY,${BOSH_ENVIRONMENT}")"
```

Comment:
Hi @miclip!

Looks like we have a failure here due to a typo. Might be leftover from the `basename` from  before:
```
bbr-pipeline-tasks-repo/tasks/bbr-backup-pas/../../scripts/export-director-metadata: line 56: syntax error near unexpected token `)'
```

Reply:
Thanks, weird that our test worked. I guess no proxy isn't that particular